services/dkg/pedersen/controller/action.go

```diff
@@ -163,19 +245,8 @@ func (a *exportInfoAction) Execute(ctx node.Context) error {
 	if err != nil {
 		return xerrors.Errorf("failed to marshal address: %v", err)
 	}
-	var pubkey kyber.Point
```

Comment:
Now the export command doesn't make sense anymore. However for debug purposes it would be useful to print out the content of DKG map so one can see all the dkg stored.